front/app/components/PostShowComponents/SharingModalContent/index.tsx

```diff
@@ -105,7 +105,7 @@ const Description = styled.p`
 const SharingWrapper = styled.div`
   flex-shrink: 0;
   width: 100%;
-  max-width: 300px;
+  max-width: 320px;
   display: flex;
```

Comment:
Added this extra width so all buttons could fit on a single row as per the design.

Reply:
Let's use a Box here?